app/components/bali/progress/component.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= tag.div(**options) do %>
+  <%= tag.progress(value: value, max: 100, style: @progress_bar_color) %>
+
+  <% if display_percentage %>
+    <%= tag.span class: 'percentage' do %>
+      <%= value %>%
```

Comment:
Instead adding a `percentage` class to use `font-size:small`. I would recommend to use `<small>` tag instead